app/harvesters/itms_harvesters/association_translator_service.rb

```diff
@@ -0,0 +1,20 @@
+# TODO
+# does not belong to here
+class ItmsHarvesters::AssociationTranslatorService
+
+  class UndefinedAssociationError < StandardError; end
+
+  # TODO
+  # locales?, better idea?
+  def self.translate(object, name)
```

Comment:
Toto uplne nechapem na co sluzi, podla mna premature abstraction - zatial by som nechal v samotnom jobe/service ako `TRANSLATIONS` hash. Keep it simple.